manifests/The Adamas Project.yaml

```
@@ -0,0 +1,13 @@
+name: The Adamas Project
+authors: toilethinges & Bereskatuket(Kestrel1110)
+homepage: https://github.com/toilethinges/Adamas-Project
+license: GPL-3.0-or-later
+version: 0.1.0
+shortDescription: Adds a very advanced race of aliens to the northeast section of the galaxy.
+description: This plugin adds a race of highly advanced Tier 2 - 3 aliens known as the Adamas to the northeast section of the galaxy, accessible through wormholes. The aliens are friendly, but they have their own versions of pirates. The Scavengers have their own galaxy that they live in, and from which they mount raids upon the Adamas. Be careful you do not get in their way, as some of their ships are armed with stolen quarg tech!
+url: https://github.com/toilethinges/Adamas-Project/archive/refs/tags/v0.1.0.zip
+iconUrl: https://raw.githubusercontent.com/toilethinges/Adamas-Project/master/icon.png
```

This should be versioned though.

```suggestion
iconUrl: https://raw.githubusercontent.com/toilethinges/Adamas-Project/v0.1.0/icon.png
```